php/helpers.php

```diff
@@ -89,6 +89,8 @@ function block_field( $name, $echo = true ) {
 		 * and then output the field with a more suitable escaping function.
 		 */
 		echo wp_kses_post( $value );
+
+		return null;
```

Comment:
This can be a breaking change for something like:

```php
$url = get_field( 'url-field' );
if ( $url ) {
        echo '<a href="' . esc_url( $url ) . '">Click here</a>'
}
```

But in that case, there would have already be unexpected output:

```
https://example.com <a href="https://example.com">Click here</a>
```

Maybe there are cases where this would break something that was working.

Reply:
Maybe something like this would be a problem:

```php
$image = get_field( 'foo-image' );
if ( $image ) {
        echo '<p>Example caption of the image</p>';
}
```

We could note in the Changelog that `block_field( 'foo' )` will not return anything anymore. So if there are conditionals that depend on it, please migrate to `block_value()`.

Reply:
I don't think that this is a breaking change worth worrying about.

Reply:
OK, thanks a lot for reviewing this.